tools/travis_script.py

```diff
@@ -209,6 +209,8 @@ def job_stm32f4dis():
                         'configs/stm32f4discovery/usbnsh/defconfig')])
 
     for buildtype in BUILDTYPES:
+        # The .bss section size is greather than the sram of STM32 board.
+        continue
```

Comment:
Interesting approach to skip the for loop. Instead, could we just return before the for loop, and leave a `TODO` comment to investigate the NuttX binary image?